ml_pipeline/requirememts.txt

```diff
@@ -0,0 +1,7 @@
+python==3.10.0
+gdal
+numpy
+rasterio
+geopandas
+pycaret
+tqdm
```

Comment:
Generally it's good practice to specify specific versions. Indeed, then when you come back to it in six months / 1 year you don't find it broken because a library upgraded without you realising! Could you specify the library versions you are using?